examples/go.mod

```diff
@@ -302,7 +302,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/pulumi/appdash v0.0.0-20231130102222-75f619a67231 // indirect
 	github.com/pulumi/esc v0.6.2 // indirect
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.73.1-0.20240202175534-6d1962d8367a // indirect
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.73.1-0.20240206184959-b82e8b2a8aff // indirect
```

Comment:
https://github.com/pulumi/pulumi-terraform-bridge/compare/6d1962d8367a...b82e8b2a8aff full changelog for the bridge changes